.github/workflows/release_to_pypi.yml

```diff
@@ -45,4 +55,64 @@ jobs:
             allowUpdates: true
             omitNameDuringUpdate: true
             makeLatest: true
-            artifacts: dist/*
\ No newline at end of file
+            artifacts: dist/*
+
+  trigger-build-aio:
```

Comment:
The trigger job is also executed when tagging for releasing PyPI. 
It is not expected.